gcc/testsuite/lib/rust.exp

```diff
@@ -181,7 +181,7 @@ proc rust_target_compile { source dest type options } {
     ## FIXME: until the compiler is made less verbose, we need to prune its output almost completely.
     # Only keep lines containing certain diagnostics so that we can check these.
     global additional_prunes
-    lappend additional_prunes "^((?!(error: |warning: )).)*$"
+    lappend additional_prunes "^((?!(error: |warning:|\[Ii\]nternal compiler error* )).)*$"
```

Comment:
That should be `"^((?!(error: |warning: |\[Ii\]nternal compiler error: )).)*$"`.  ;-)

Reply:
nice catch, sorry, quick copy/paste from the dg-ice handling code!